public/js/app/models/Post.js

```diff
@@ -23,7 +24,15 @@ define(["config",
     isHidden: DS.attr('boolean'),
 
     timeline: DS.belongsTo('timeline'),
-
+    didLoad: function(){
+      setInterval(
+        Ember.run.bind(this, this.updateTime)
+        , 1000)
```

Comment:
once a second? I'm not sure we need that kind of granularity here.
10 seconds should be just fine

(just asking what other ppl think)